Makefile

```diff
@@ -86,7 +54,7 @@ pylint:
 	@pylint --rcfile utils/pylintrc sphinx
 
 reindent:
-	@$(PYTHON) utils/reindent.py -r -n .
+	@echo "This target no longer does anything and will be removed imminently"
```

Comment:
+1. I'd never use this script.
But we need to announce the deprecation to sphinx-devel list before merging this.

Reply:
So just to confirm, you're OK with this approach?

Reply:
Yes, I'm okay to drop whole of `reindent` target if nobody opposes.

Reply:
OK, I've moved this to a separate PR so we don't block this